src/store/index.js

```diff
@@ -0,0 +1,16 @@
+import Vue from "vue";
+import Vuex from "vuex";
+import actions from "./actions";
+import mutations from "./mutations";
+
+Vue.use(Vuex);
+
+const store = new Vuex.Store({
+  actions,
+  mutations,
+  state: {
+    user: null,
+  },
+});
+
```

Comment:
Creates a new store, and says that the store will contain actions and mutations

It also has a "state" object which allows us to set default states. In this case, we set the default user to null, which means there is no user until some other part of the app says otherwise.

This also means that the store will always know the current state of the user so it can share that knowledge elsewhere

I'm not sure what an example of a different value of user would be. Maybe the username or email address?